### PR TITLE
Add Ethereum router

### PR DESCRIPTION
**Pull request by claravanstaden**
Add a separate router to Ethereum, instead of reusing the existing Rococo router.

Resolves: [SNO-731](https://linear.app/snowfork/issue/SNO-731)

### CODE REVIEW

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs [hunk 1]
```diff
@@ -874,7 +874,36 @@ impl pallet_xcm_bridge_hub_router::Config<ToRococoXcmRouterInstance> for Runtime
...
+			Runtime,
+		>;
+
+	type ByteFee = xcm_config::bridging::XcmBridgeHubRouterByteFee;
+	type FeeAsset = xcm_config::bridging::XcmBridgeHubRouterFeeAssetId;
+}
+
+pub type ToEthereumXcmRouterInstance = pallet_assets::Instance3;
+impl pallet_xcm_bridge_hub_router::Config<ToEthereumXcmRouterInstance> for Runtime {
+	type WeightInfo = weights::pallet_xcm_bridge_hub_router_to_rococo::WeightInfo<Runtime>; // TODO do for ethereum
```
Comment (claravanstaden) [inline cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs:903]: Not sure if we should add our own weights file here - there doesn't seem to be a difference in the xcm router pallet weights (i.e. Rococo and Wococo xcm router pallet weights are generated with the same command)

**Comment by claravanstaden** [comment]
Tested:

- [x] register token
- [x] send token
- [x] transfer token
Routing from `Ethereum -> Bridge Hub -> Asset Hub` and `Asset Hub -> Bridge Hub -> Ethereum` works with the new router config.

File: cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs (hunk 1 above)
Comment (vgeddes) [reply cumulus/parachains/runtimes/assets/asset-hub-rococo/src/lib.rs:903]: yeah just the copy the file and rename it appropriately, I'd say.

**Review by vgeddes — APPROVED** [review]

**Review by alistair-singh — APPROVED** [review]

**Comment by claravanstaden** [comment]
Closed in favour of https://github.com/Snowfork/polkadot-sdk/pull/16.

**Pull request closed without merging**